.changeset/strange-fireants-burn.md

```diff
@@ -0,0 +1,14 @@
+---
+"@mysten/wallet-adapter-wallet-standard": minor
+"@mysten/wallet-adapter-unsafe-burner": minor
+"@mysten/wallet-adapter-base": minor
+"@mysten/wallet-kit-core": minor
+"@mysten/wallet-standard": minor
+"@mysten/wallet-kit": minor
+---
+
+wallet-standard: changes sui:signAndExecuteTransaction and sui:signTransaction features to support account and chain options
```

Comment:
(don't worry about doing this now) in the future you should just create multiple changesets so that these release messages are more logically grouped.